src/components/searchbar/SearchBar.jsx

```diff
@@ -2,26 +2,30 @@ import { updateUserInput } from './searchBarSlice';
 import { useDispatch, useSelector } from "react-redux";
 import { updateSearchTerm } from "../reddit/RedditSlice";
 import { selectUserInput } from "./searchBarSlice";
+import React from 'react';
 
 const SearchBar = () => {
-    const dispatch = useDispatch()
-    const userInput = useSelector(selectUserInput)
+    const dispatch = useDispatch();
+    const userInput = useSelector(selectUserInput);
+    const [value, setValue] = React.useState('');
```

Comment:
you already have userInput available here. you don't need to have another state to check

Reply:
got it